pkgs/by-name/ni/niri/package.nix

```diff
@@ -52,8 +43,8 @@ rustPlatform.buildRustPackage rec {
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
-      "smithay-0.3.0" = "sha256-nSM7LukWHO2n2eWz5ipFNkTCYDvx/VvPXnKVngJFU0U=";
       "libspa-0.8.0" = "sha256-kp5x5QhmgEqCrt7xDRfMFGoTK5IXOuvW2yOW02B8Ftk=";
+      "smithay-0.3.0" = "sha256-nSM7LukWHO2n2eWz5ipFNkTCYDvx/VvPXnKVngJFU0U=";
```

Comment:
Why reorder these? They're sorted now, I guess, but smithay is The Main Dependency and therefore deserves top billing.

Reply:
I think alphabetical makes the most sense

Reply:
I just ran the update script, which orders them automatically.

Reply:
yeah, wasn't looking to discuss this extensively (see: approved before I got any response), just felt out of place. update script outputting it is a perfectly valid justification; I just thought it was odd if this was *manually* reordered.